resolver/dns/go19.go

```diff
@@ -0,0 +1,43 @@
+// +build go1.9
+
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package dns
+
+import (
+	"net"
+	"strings"
+
+	"golang.org/x/net/context"
+)
+
+var customAuthorityResolver = func(authority string) (netResolver, error) {
+	if _, _, err := net.SplitHostPort(authority); err != nil && strings.Contains(err.Error(), "missing port in address") {
+		authority = authority + ":53"
+	}
+
+	return &go18upResolver{
```

Comment:
the `&go18upResolver` as commented above, is unnecessary. Just return `&net.Resolver{...}`